docker/patched-star/Dockerfile

```diff
@@ -7,8 +7,17 @@ LABEL maintainer="Ambrose J. Carr <acarr@broadinstitute.org>" \
   website="https://github.com/alexdobin/STAR"
 
 RUN apt update && apt install -y \
+  g++ \
```

Comment:
I assume these are all dependencies of samtools. If that's the case, can you install star before we install these dependencies, just so it's clear they're for samtools?

Reply:
Yes, most of them are dependencies of samtools 1.6, I will re-organize the order to make it clearer for this docker and other necessary dockerfiles. Thanks for this suggestion!

Reply:
This docker or STAR  probably is also obsoleted. Ambrose should give a look later if we can build STAR docker from latest release.